utilities/rpp-unittests/HOST_NEW/Tensor_host_pkd3.cpp

```diff
@@ -457,10 +567,11 @@ int main(int argc, char **argv)
 
         Rpp32f alpha[images];
         Rpp32f beta[images];
+
         for (i = 0; i < images; i++)
         {
-            alpha[i] = 1.75;
-            beta[i] = 50;
+            alpha[i] = pBrightness[0];
```

Comment:
After input validation passes, the user param can be used directly. Even if one of the params fails validation, force the user to enter correct params. If no params are entered by the user at all, only then the default params should run.